osf/external/gravy_valet/request_helpers.py

```diff
@@ -257,8 +257,10 @@ def get_gv_citation_url_list_for_project(auth, addon_short_name, project, reques
 def _invoke_gv_citation_operation_invocations(auth, addon, project, list_id):
     data = {
         'attributes': {
-            'operation_name': 'list_root_collections',
-            'operation_kwargs': {},
+            'operation_name': 'list_collection_items',
+            'operation_kwargs': {
+                'collection_id': addon['attributes']['root_folder'] if list_id == 'ROOT' else list_id
```

Comment:
I've noticed the issue when actual root items would be fetched instead if child items of configured root. This resolves it
@adlius fyi